splitio/client/client.py

```diff
@@ -115,8 +115,9 @@ def _make_evaluation(self, key, feature, attributes, method_name, metric_name):
             )
             self._record_stats([(impression, attributes)], start, metric_name, method_name)
             return result['treatment'], result['configurations']
-        except Exception:  # pylint: disable=broad-except
+        except Exception as e:  # pylint: disable=broad-except
             _LOGGER.error('Error getting treatment for feature')
+            _LOGGER.error(str(e))
```

Comment:
do we need this?

Reply:
I think so, I had to add this line to know why the treatment evaluation is failing, we are not showing it in the exception block